model/ConsumerService.php

```diff
@@ -97,7 +97,7 @@ public function getAvailableEntryPoints()
     {
         /** @var DefaultUrlService $urlService */
         $urlService = $this->getServiceManager()->get(DefaultUrlService::SERVICE_ID);
-        $routes = $urlService->getOptions();
+        $routes = $urlService->getOption('routes');
```

Comment:
Hi, could u add this method into defautlUrlService and also in NccerUrlService to be able to easily override it please?